app-tracking-protection/vpn-impl/src/main/java/com/duckduckgo/mobile/android/vpn/breakage/ReportBreakageCategorySingleChoiceActivity.kt

```diff
@@ -147,7 +149,7 @@ class ReportBreakageCategorySingleChoiceActivity : DuckDuckGoActivity() {
                     reportedFrom = brokenApp.launchFrom,
                     appName = brokenApp.appName,
                     appPackageId = brokenApp.appPackageId,
-                    description = binding.appBreakageFormFeedbackInput.text.toString(),
+                    description = binding.appBreakageFormFeedbackInput.text,
                     category = viewModel.viewState.value.categorySelected.toString(),
```

Comment:
@aitorvs I think we should` viewModel.viewState.value.categorySelected.key` either use or implement `toString()` for `AppBreakageCategory` similar to how we were doing previously.

With the current changes, we are sending: `breakageCategory=AppBreakageCategory(key=connection, description=La aplicación no tiene conexión)` in the  pixel instead of just `breakageCategory=connection` like before.

Reply:
Oh yeah, missed that. Will fix

Reply:
Good catch, totally missed it. will fix